data/mods/Aftershock/recipes/deconstruction/bionic_deconstruction.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "result": "burnt_out_bionic",
+    "type": "uncraft",
+    "skill_used": "electronics",
+    "difficulty": 7,
+    "time": "10 m",
+    "using": [ [ "soldering_standard", 10 ], [ "welding_standard", 10 ] ],
+    "qualities": [ { "id": "BIONIC_ASSEMBLY", "level": 1 } ],
+    "components": [
```

Comment:
Will these components be filthy coming from a filthy burnt out bionic?  Do you want them to need only a wash or do you want them to also run through an autoclave?

Reply:
No they come out clean and ready to use (as designed by me)